lib/plugins/aws/deploy/lib/initializeResources.js

```diff
@@ -1,63 +1,27 @@
 'use strict';
 
-const _ = require('lodash');
 const path = require('path');
 const BbPromise = require('bluebird');
 
 module.exports = {
   initializeResources() {
-    const coreCFTemplate = this.serverless.utils.readFileSync(
+    // save the custom resources (e.g. Resources, Outputs etc.) the user has added
+    // so that they can be merged in later on
+    this.serverless.service.customProviderResources = {};
+
+    if (this.serverless.service.resources && this.serverless.service.resources.Resources) {
```

Comment:
I don't think the second check is necessary. if resources.Resources doesn't exist it doesn't really matter, but if people only have a description or Outputs they wouldn't be copied